src/pm/hydra/ui/mpich/utils.c

```diff
@@ -440,6 +440,32 @@ static HYD_status profile_fn(char *arg, char ***argv)
     goto fn_exit;
 }
 
+static void output_from_help_fn(void)
+{
+    printf("\n");
+    printf("-show-rank: only show the output from this rank\n\n");
```

Comment:
Need to update this line to have the new option?

```suggestion
    printf("-output-from: only show the output from this rank\n\n");
```

Reply:
Yep, good catch